website/docs/cloud-docs/workspaces/creating.mdx

```diff
@@ -34,48 +31,51 @@ New workspaces can be created by teams with permission to manage workspaces. ([M
+1. Click **Workspaces** in the top navigation bar to view a list of the workspaces within your organization.
+1. Click **+ New Workspace**. The **Create a new Workspace** page appears.
+1. Choose a workflow type: **Version control**, **CLI-driven**, or **API-driven**.
+1. (Version control only) Choose an existing version control provider or configure a new one. Refer to [Connecting VCS Providers](/cloud-docs/vcs) for more details.
+1. (Version control only) Choose a repository from the filterable list. The **Configure settings** page appears.
 
-To create a new workspace:
+-> **Note:** Depending on your provider, you may need to change which account's repositories are shown. If the repository is not listed, scroll to the bottom of the list and enter its ID in the text field.
 
-1. [Navigate to the workspace list](/cloud-docs/workspaces/#listing-and-filtering-workspaces) and click the "+ New Workspace" button (near the top of the page).
+1. Enter a **Workspace Name**. This defaults to the repository name, if applicable. The name must be unique within the organization and can include letters, numbers, dashes (`-`), and underscores (`_`). Refer to our [workspace naming recommendations](/cloud-docs/workspaces/naming).
+1. Add an optional **Description** that will appear at the top of the workspace in the Terraform Cloud UI.
 
-1. On the first screen, choose your VCS provider (or choose "No VCS connection").
+1. (Version control only) Open **Advanced options** to optionally configure the following additional settings:
+   - [Terraform Working Directory][workdir]
+   - [Automatic Run Triggering][trigger]
+   - [VCS branch][branch]
+   - [Include submodules on clone][submodules]
 
-   -> **Note:** If you haven't added a VCS provider for your organization yet, choosing one here will prompt you to configure it. See [Connecting VCS Providers](/cloud-docs/vcs) for more information.
+1. Click **Create workspace**.
 
-1. On the second screen, choose a repository from the filterable list. This screen is skipped if you chose "No VCS connection".
+For CLI and API-driven workspaces, the workspace overview page appears. For version control workspaces, the **Configure Terraform variables** page appears.
 
-   Some VCS providers limit the list's size. If a repository isn't listed, you can still choose it by name; scroll to the bottom of the list and enter its ID in the text field.
+### Configure Terraform Variables (Version Control Only)
 
-   -> **Note:** For some VCS providers, this list includes a drop-down menu for changing which account's repositories are shown. Other providers combine all available accounts into a single list.
+After you create a new workspace from a version control repository, Terraform Cloud scans its configuration files for [Terraform variables](/cloud-docs/workspaces/variables#terraform-variables) without default values and displays them on the page. Terraform cannot perform successful runs until you set values for these variables.
```

Comment:
Might be worth pointing out that variables that are in varsets applied to the workspace will also not show up in the form.

```suggestion
After you create a new workspace from a version control repository, Terraform Cloud scans its configuration files for [Terraform variables](/cloud-docs/workspaces/variables#terraform-variables) that are not present in applicable varsets or do not have default values and displays them on the page. Terraform cannot perform successful runs until you set values for these variables.
```

Reply:
Might also be worth mentioning that this does not include environment variables either?

Reply:
I thought it would be unnecessary because Terraform variables and environment variables are different types of variables (it's clarified in the Terraform variable link). If we say Terraform variable, will it be clear that we mean Terraform variables, not any type of variable? I thought it would be but totally open to changing.

Reply:
Oooo I didn't think of varsets - can those be set in a configuration file and we just ignore it during ingress?